src/test/resources/test_scripts/stages-with-wait.groovy

```diff
@@ -1,7 +1,7 @@
 node {
    stage 'Stage 1'
-   sh 'sleep 3; echo `date` Stage 1;'
-   sh 'sleep 3; echo `date` Stage 1;'
+   sh 'sleep 6; echo `date` Stage 1a we need some time here for slower ath hosts;'
```

Comment:
this doesn't seem like a good solution. add an input step instead, maybe? something else that could cause the test to pause - a wait for condition type of thing that the test could set?